tfx/components/example_validator/component_test.py

```diff
@@ -31,10 +31,21 @@ def testConstruct(self):
     statistics_artifact = standard_artifacts.ExampleStatistics()
     statistics_artifact.split_names = artifact_utils.encode_split_names(
         ['eval'])
+    example_validator = component.ExampleValidator(
+        statistics=channel_utils.as_channel([statistics_artifact]),
+        schema=channel_utils.as_channel([standard_artifacts.Schema()]))
+    self.assertEqual(standard_artifacts.ExampleAnomalies.TYPE_NAME,
+                     example_validator.outputs['anomalies'].type_name)
+
+  def testConstructWithExcludeSplits(self):
+    statistics_artifact = standard_artifacts.ExampleStatistics()
+    statistics_artifact.split_names = artifact_utils.encode_split_names(
+        ['train', 'eval'])
+    exclude_splits = ['train']
     example_validator = component.ExampleValidator(
         statistics=channel_utils.as_channel([statistics_artifact]),
         schema=channel_utils.as_channel([standard_artifacts.Schema()]),
-    )
+        exclude_splits=exclude_splits)
```

Comment:
you can just add it to a existing test, it's trivial change

Reply:
got it, i updated this